oms-hypervReplica-solution/azuredeploy.json

```diff
@@ -0,0 +1,650 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.1.0",
+  "parameters": {
+    "workspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of your OMS Log Analytics workspace."
+      }
+    },
+    "workspaceRegionId": {
+      "type": "string",
+      "allowedValues": [
+        "East US",
+        "West Europe",
+        "Southeast Asia",
+        "Australia Southeast",
+        "West Central US"
+      ],
+      "metadata": {
+        "description": "The region of your OMS Log Analytics workspace."
+      },
+      "defaultValue": "West Europe"
+    },
+    "accountName": {
```

Comment:
I had initially named the parameters like you suggest but based on the parameter names in the Microsoft document below I changed them. What would the best practise be here, paramater names that better describe the parameter or follow the documentation?

https://docs.microsoft.com/en-us/azure/operations-management-suite/operations-management-suite-solutions-solution-file